xbmc/contrib/kissfft/_kiss_fft_guts.h

```diff
@@ -1,16 +1,10 @@
 /*
-Copyright (c) 2003-2010, Mark Borgerding
-
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
-
-    * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
-    * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-    * Neither the author nor the names of any contributors may be used to endorse or promote products derived from this software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-*/
+ *  Copyright (c) 2003-2010, Mark Borgerding
+ *  All rights reserved.
+ *
+ *  SPDX-License-Identifier: BSD-3-Clause
+ *  See LICENSES/README.md for more information.
+ */
 
```

Comment:
@mborgerding, sorry to bother you. I need to know if you're OK with ^^.

It's part of the effort to convert Kodi to the **[SPDX license format](https://spdx.org/licenses/)**. Complete set of changes can be seen **[here](https://github.com/hudokkow/xbmc/pull/4)**, if you want some background.

Assuming you're OK with it, would you accept a pull request to convert `kissfft` to the SPDX license format? I already have **[some work done](https://github.com/hudokkow/kissfft/commit/ada7bfae3399e42ca2c89c3c42187425063007f0)**.

Cheers!

Reply:
I'm just catching up to speed on the SPDX effort.  I hadn't heard of it before now.  I think that is a great idea: a step toward solving licensing fatigue. I'd be happy to see your pull request.

Reply:
Great! :+1: I'll open a PR in the next few days.